### ISSUE TITLE
Enhance notifications

### ISSUE THREAD

**Issue by Lacro59** [mtkennerly/ludusavi-playnite#3]
I tested the all game backup and I've a suggestion.
The notification contains only the total of game backed up and there is any action on click it.
It was super to have a action for show the list of game backed up.
I suppose that the same system for all back up by lot.
![notification](https://user-images.githubusercontent.com/124054/88898617-4333bb80-d24d-11ea-9c0b-994a348e92a3.png)

**Comment by mtkennerly** [comment]
That's a great idea! I didn't even realize that was possible, but I see the docs for `ActivationAction` now. I'll definitely add this.

**Comment by mtkennerly** [comment]
This is implemented now. I would have liked to use a normal Playnite dialog, but they can't scroll and they'll go off the screen if you have too many games, so I used a custom webview. It's not super pretty, but it works 🙃 The default background and text color were both black, so I just changed the text to white (and made it sans-serif).

If any games aren't processed, you'll see `[IGNORED]` or `[FAILED]` by their names.

> ![image](https://user-images.githubusercontent.com/4934192/89180577-0b37bb80-d560-11ea-8fc0-1163ef25014a.png)
